python/lsst/ts/hexrotcomm/base_csc.py

```diff
@@ -371,7 +371,7 @@ async def wait_controller_state(
 
         Parameters
         ----------
-        state : `lsst.idl.enums.MTRotator.ControllerState`
+        state : `lsst.ts.xml.enums.MTRotator.ControllerState`
```

Comment:
type can be `ControllerState` only

Reply:
Fixed in the commit: [aca6aed](https://github.com/lsst-ts/ts_hexrotcomm/pull/68/commits/aca6aed921cb2791cad6c4aa62574686dcb127c8). Thanks!